LibGit2Sharp/LibGit2Sharp.csproj

```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="[2.0.305]" PrivateAssets="none" />
+    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.312" />
```

Comment:
We probably need to reference our package instead. Also might require adding a proper NuGet package source.

Reply:
Updated the reference to Mendix.LibGit2Sharp.NativeBinaries version 1.101.1

Reply:
increased the build no to 1.101.4 to test the package.
i will revert this once github clears the packages